packages/babel-plugin-proposal-pattern-matching/src/index.js

```diff
@@ -1,27 +1,24 @@
 import syntaxPatternMatching from "@babel/plugin-syntax-pattern-matching";
+import template from "@babel/template";
```

Comment:
and here you can do `import { template } from "@babel/core";` since template was removed as a dep

Reply:
@hzoo got it.